freqtrade/persistence/models.py

```diff
@@ -569,7 +569,7 @@ def update(self, order: Dict) -> None:
                 payment = "BUY" if self.is_short else "SELL"
                 # * On margin shorts, you buy a little bit more than the amount (amount + interest)
                 logger.info(f'{order_type.upper()}_{payment} has been fulfilled for {self}.')
-            # TODO-lev: Double check this
+            # TODO-lev: Is anything else needed here?
```

Comment:
@xmatthias I'm asking you this, do we need to add anything here?

Reply:
I would think this has to do with the comment 2 lines above here - but i don't think there's much we can do exactly here - other than obviously verifying that we paid back everything we entered for.

Reply:
We only need to do that on margin trading anyway